pages/index.vue

```diff
@@ -34,6 +35,30 @@ export default {
       articles,
       categories
     }
+  },
+  computed: {
+    isDarkMode () {
+      if (process.browser) {
+        return window.matchMedia('(prefers-color-scheme: dark)').matches
+      }
+      return false
+    }
+  },
+  head () {
+    return {
+      title: 'BitcartCC Blog',
+      titleTemplate: '%s - BitcartCC Official Blog',
+      meta: [
+        { charset: 'utf-8' },
+        { name: 'viewport', content: 'width=device-width, initial-scale=1' },
+        {
+          hid: 'description',
+          name: 'description',
+          content: 'BitcartCC Official Blog'
+        }
+      ],
+      link: [{ rel: 'icon', type: 'image/x-icon', href: this.isDarkMode ? '/favicon_dark.ico' : '/favicon.ico' }]
```

Comment:
Interesting idea!
But what about using a favicon with white background, which should be visible in dark and light modes perfectly instead?

Reply:
Removed the code since favicon with white background is enough